packages/tools/pie-icons-configs/__tests__/configs.test.js

```diff
@@ -110,126 +112,136 @@ describe('validateGetRegularIconSize', () => {
+        describe('svgClasses parameter', () => {
+            it('returns an object that contains a class attr with the expected classes', () => {
+                const received = getSvgProps('icon-test', null, null, regularIconSizeName);
+
+                expect(received).toHaveProperty('class');
+                expect(received.class).toEqual('icon-test');
+            });
+        });
 
-        describe('when provided with valid parameters', () => {
-            it('returns an object that contains a class and width and height properties', () => {
-                const received = getSvgProps('pie-icon pie-icon--app-order', null, regularIconSizeDefault, iconName);
+        describe('staticClasses parameter', () => {
+            it('returns an object that contains a class attr with the expected classes', () => {
+                const received = getSvgProps('icon-test', 'FOO BAR', null, regularIconSizeName);
 
                 expect(received).toHaveProperty('class');
-                expect(received).toHaveProperty('width');
-                expect(received).toHaveProperty('height');
+                expect(received.class).toEqual('icon-test FOO BAR');
             });
+        });
 
-            it('has expected width and height values', () => {
-                const received = getSvgProps('pie-icon pie-icon--app-order', null, regularIconSizeDefault, iconName);
+        describe('valid iconSizeValue parameter', () => {
+            it('returns an object that contains a width and height attr with the correct values', () => {
+                const received = getSvgProps('icon-test', null, regularIconSizeDefault, regularIconSizeName);
                 const expectedRegularIconSizeValue = sizeToValueMap[regularIconSizeDefault];
 
+                expect(received).toHaveProperty('width');
+                expect(received).toHaveProperty('height');
                 expect(received.width).toEqual(expectedRegularIconSizeValue);
                 expect(received.height).toEqual(expectedRegularIconSizeValue);
             });
+        });
+    });
 
-            describe('when provided with a staticClasses parameter', () => {
-                it('returns an object with the expected classes', () => {
-                    const received = getSvgProps('pie-icon pie-icon--app-order', 'FOO BAR', regularIconSizeDefault, iconName);
+    describe('when provided with invalid iconSizeValue parameter', () => {
+        const invalidIconSizeValue = 'n';
 
-                    expect(received.class).toEqual(expect.stringContaining('pie-icon'));
-                    expect(received.class).toEqual(expect.stringContaining('pie-icon--app-order'));
-                    expect(received.class).toEqual(expect.stringContaining('FOO BAR'));
-                });
-            });
+        it('returns an object with valid width and height properties', () => {
+            const errorMock = vi.fn();
+            console.error = errorMock;
```

Comment:
It's probably better practice if we spy on this and mock the implementation like in the other tests

Reply:
And I know it's not used in every test, but perhaps it's used enough now that we can justify pulling it out into a `beforeEach` to set up the mock and an `afterEach` to restore it